packages/experimental/src/experimental-list/experimental-list.tsx

```diff
@@ -18,14 +18,14 @@ export type ListProps = {
 
 export const ExperimentalList: React.FC< ListProps > = ( {
 	children,
-	listType = 'ul',
```

Comment:
This change looks random, but when using args with experimental list, it gives "undefined" as a possible prop value, if you choose that it crashes the component currently, by falling back to `ul` later it means if someone passed a falsy value the component won't crash